### PR TITLE
ENH: Adds `left_trim_len` parameter

### LINKED ISSUE #51: Expose `--left-trim-length` in plugin
Looks like this functionality was recently merged in Deblur: https://github.com/biocore/deblur/pull/156

### PR DESCRIPTION
**Pull request by thermokarst**
Fixes #51 

No tests at the moment, not sure if adding any will be particularly useful, either.

As a side note - this plugin could use some TLC - there are a handful of things that could be cleaned up here.

### CODE REVIEW

**Comment by thermokarst** [comment]
@ChrisKeefe, @turanoo, @nbokulich, interested in reviewing?

**Comment by thermokarst** [comment]
@nbokulich - as requested....

File: q2_deblur/tests/test_denoise.py [hunk 1]
```diff
@@ -101,8 +101,17 @@ def test_defaults(self):
         self.assertEqual(list(stats.columns), STATS_HEADER[1:])
         self.assertEqual(len(stats), 0)
 
+    def test_left_trim_len(self):
+        obs_tab, rep_seqs, stats = denoise_16S(self.demux_seqs, 110,
+                                               left_trim_len=10)
+
+        self.assertEqual(len(obs_tab.ids(axis='sample')), 16)
+        self.assertEqual(len(obs_tab.ids(axis='observation')), 20)
+        self.assertEqual(len(list(rep_seqs)), 20)
```
Comment (nbokulich) [inline q2_deblur/tests/test_denoise.py]: what about making sure that all sequences are trimmed to the correct length?

Comment (thermokarst) [reply q2_deblur/tests/test_denoise.py]: Are we in the business of testing deblur now?

Comment (thermokarst) [reply q2_deblur/tests/test_denoise.py]: Note, that was sarcasm!

Comment (nbokulich) [reply q2_deblur/tests/test_denoise.py]: good point... unnecessary.

**Pull request merged**